Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># CMake generated files\nCMakeFiles/\nCMakeCache.txt\ncmake_install.cmake\nMakefile\n\n.vscode/\n\nsrc/output/\n\n\n# Compiled object files, libraries, executables\n*.obj\n*.o\n*.a\n*.lib\n*.so\n*.dll\n*.exe\n\n# Temporary files\n*.tmp\n*.log\n*.bak\n*.swp\n\n# Directories\nbuild/
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision ee462d4d8de85750884d0c947e0d0cbeae8849a9)
+++ b/.gitignore	(date 1734562353845)
@@ -4,6 +4,9 @@
 cmake_install.cmake
 Makefile
 
+.idea/
+cmake-build-debug/
+
 .vscode/
 
 src/output/
